src/typo3/customize.md

```diff
@@ -19,6 +21,17 @@ client.html.catalog.filter.default.button = 1
     client.html.catalog.lists.standard.template-body = catalog/lists/body-mytemplate
     ```
 
+!!! important
+    The *ai-typo3* extension brings even more configuration options along. Due to the way the configuration section of this documentaion is generated, these options do not show up currently. Please have a look at the source files at `<typo3-root>/ext/ai-typo3/lib/custom/src/MShop/Customer/Manager/Typo3.php` and `<typo3-root>/ext/ai-typo3/lib/custom/src/MShop/Customer/Manager/Group/Typo3.php` for detailed information. The options in question all start with `mshop/customer/manager/typo3/` or  `mshop/customer/manager/group/typo3/`. The 2 most common options needed to configure the `Users` folder (where customers are saved to on account creation) are:
```

Comment:
We should rather add the config settings to the docs instead of stateing that they are not there

Reply:
As for "taxflag" and "precision" (2 vars I am aware of that also don't appear in the documentation) I didn't find the reason why they are not listed. The comments (from where your script fetches the contents, right?) seem to be written correctly... and build.xml seems to incorporate all of mshop and madmin.
As for the TYPO3 configuration option: they come from a different ext (ai-typo3). As far as I understand the current documentation "only" covers aimeos-core.